specs/phase0/beacon-chain.md

```diff
@@ -1313,7 +1314,9 @@ def get_attestation_deltas(state: BeaconState) -> Tuple[Sequence[Gwei], Sequence
         attesting_balance = get_total_balance(state, unslashed_attesting_indices)
         for index in eligible_validator_indices:
             if index in unslashed_attesting_indices:
-                rewards[index] += get_base_reward(state, index) * attesting_balance // total_balance
+                increment = REWARD_OVERFLOW_INCREMENT  # Factored out from reward numerator to avoid uint64 overflow
+                reward_numerator = get_base_reward(state, index) // increment * attesting_balance
```

Comment:
since attesting balance is a total of 16-32 ETHs, it's precision is lower. Can we not divide that by the increment, then multiply again later, to keep more precision in the resulting reward?

Reply:
ypu good call